2-patterns_integrations/src/main/java/com/epam/dhontar/aqamp/utils/enums/Constants.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.dhontar.aqamp.utils.enums;
+
+public enum Constants {
```

Comment:
I guess it would be better to rename this enum into "TestRailCredentials"

Reply:
Totaly agree. Done